soc/arm/nxp_kinetis/k2x/Kconfig.defconfig.mk22f12

```diff
@@ -0,0 +1,107 @@
+# Kconfig - FSL FRDM K22F platform configuration options
+#
+# Copyright (c) 2018 Prevas A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_MK22F51212
+
+config SOC
+	string
+	default "mk22f51212"
+
+if ADC
+
+config ADC_MCUX_ADC16
+	default y
+
+endif # ADC
+
+if CLOCK_CONTROL
+
+config CLOCK_CONTROL_MCUX_SIM
+	default y
+
+config SMC_MCUX
+	default y
+
+endif # CLOCK_CONTROL
+
+if PINMUX
+
+config PINMUX_MCUX
+	default y
+
+endif # PINMUX
+
+config GPIO
+	default y
+
+if GPIO
+
+config GPIO_MCUX
+	default y
+
+endif # GPIO
+
+if I2C
+
+config I2C_MCUX
+	default y
+
+endif # I2C
+
+if PWM
+
+config PWM_MCUX_FTM
+	default y
+
+endif # PWM
+
+config SPI
+	default n
```

Comment:
These two lines can be removed. Bool configs are default n unless explicitly set otherwise.